packages/react/src/use-chat.ui.test.tsx

```diff
@@ -1855,3 +1855,73 @@ describe('initialMessages', () => {
     });
   });
 });
+
+describe('resume ongoing stream and return assistant message', () => {
+  const controller = new TestResponseController();
+
+  setupTestComponent(
+    () => {
+      const { messages, status, experimental_resume } = useChat({
+        id: '123',
+        initialMessages: [{ id: 'msg_123', role: 'user', content: 'hi' }],
+      });
+
+      useEffect(() => {
+        experimental_resume();
+      }, []);
```

Comment:
pls fix eslint

Reply:
(also relevant for examples)

Reply:
add a comment to ignore that eslint error since the `useEffect` only needs to run once

Reply:
lint error still there?